packages/syft/src/syft/service/request/request.py

```diff
@@ -287,7 +286,7 @@ def apply(self, context: AuthedServiceContext) -> Result[SyftSuccess, SyftError]
             # by default change status is not applied
             change_status = ChangeStatus(change_id=change.id, applied=False)
             result = change.apply(context=change_context)
-            if result.is_err():
+            if isinstance(result, SyftError):
```

Comment:
similarly here, apply is returning the result of _run, which currently are returning SyftSuccess or SyftError wrapping in `Ok` or `Err` respectively.

Reply:
Resolved by [dd43d68](https://github.com/OpenMined/PySyft/pull/7897/commits/dd43d68b9c30fa5cab8ced182881510481381ea6)